frontend/packages/console-shared/src/components/editor/yaml-editor-utils.ts

```diff
@@ -147,6 +148,42 @@ export const registerYAMLHover = (languageID, monaco, m2p, p2m, createDocument,
   });
 };
 
+export const registerYAMLFolding = (languageID, monaco, createDocument) => {
+  monaco.languages.registerFoldingRangeProvider(languageID, {
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    provideFoldingRanges(model, context, token) {
```

Comment:
Will this disable all of the other folding ranges so I can't fold sections like `metadata`, `spec`, or `status`?

Reply:
Yeah, I just realized this. I have another alternative way that would keep all the folding sections + be able to fold managedFields. I'm just making those changes now

Reply:
I've made the changes and now the managedFields property is collapsed and all the other folding sections show up as well!